config/src/fuzz.rs

```diff
@@ -60,6 +60,9 @@ impl InlineConfigParser for FuzzConfig {
             match key.as_str() {
                 "runs" => conf_clone.runs = parse_config_u32(key, value)?,
                 "max-test-rejects" => conf_clone.max_test_rejects = parse_config_u32(key, value)?,
+                "dictionary-weight" => {
+                    conf_clone.dictionary.dictionary_weight = parse_config_u32(key, value)?
```

Comment:
> should we restrict the possible input range?

yea if we can restrict this to 0-100 which are the only valid values and error otherwise that would be great. (I'm not sure if floats are allowed or ints only)